test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -50,6 +50,16 @@ Feature: Status Desktop Chat
          Then the user can delete the message at index 0
          Then the last message is not the random message
 
+    Scenario: User can clear chat history
+        When user joins chat room test
+        Then user is able to send chat message
+        | message  			    |
+		| Hello    			    |
+		| How are you    		|
+		| I am from status   	|
+		| tell me how you do?   |
+		Then The user is able to clear chat history
```

Comment:
**Suggestion:**
Maybe we should split the action and validation in 2 different statements:
```
When the user clears chat history
Then the chat is cleared
```
And in the validation, apart from validating the `chat_log.count`, maybe we can add, as well, that the last message is not displayed.

WDYT?